tests/test_lc_l4_natural_surface.py

```diff
@@ -179,9 +179,7 @@ def test_ns():
     veg_cover = l4_veg_cover.canopyco_veg_con(xx, stats_l4.veg_threshold)
```

Comment:
could you mock the veg_threshold? `stats_l4` is created purely for getting the threshold in this instance. Ideally we should have all those thresholds in config yaml. We should avoid accidentally use the default config in the code, better raise error if not getting them from config yaml.

Reply:
This is addressed.